controllers:
  chassis_controller:
    type: rm_chassis_controllers/SentryController
    publish_rate: 100
    enable_odom_tf: true
    wheel_radius: 0.06
    wheel:
      joint: "drive_wheel_joint"
      pid: { p: 0.3, i: 0, d: 0.0, i_max: 0.0, i_min: 0.0, antiwindup: true, publish_state: true }
    catapult:
      joint: "catapult_joint"
      pid: { p: 50, i: 0, d: 1.5, i_max: 0.0, i_min: 0.0, antiwindup: true, publish_state: true }
      catapult_angle: 1.57
      velocity_coefficient: 0.5
      lock_duration: 1.0
    twist_covariance_diagonal: [ 0.001, 0.001, 0.001, 0.001, 0.001, 0.001 ]
    power:
      effort_coeff: 0.01
      vel_coeff: 0.0048
      power_offset: 0
    timeout: 0.1
